tests/tracing/bug01732-001.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Test for bug #1732 - Trace flamegraph format output for time
+--INI--
+xdebug.mode=trace
+xdebug.start_with_request=0
+xdebug.trace_format=3
+xdebug.dump_globals=0
+xdebug.collect_return=0
+xdebug.collect_assignments=0
+xdebug.force_error_reporting=0
```

Comment:
Neither of these 4 is useful, so please just remove them.

Reply:
Oh, that was a copy paste from one computerized trace output bugfix test, OK I'll drop those.